pkg/manager/member/tikv_member_manager.go

```diff
@@ -762,6 +762,11 @@ func (m *tikvMemberManager) syncTiKVClusterStatus(tc *v1alpha1.TidbCluster, set
 		if err = endEvictLeader(m.deps, tc, helper.GetMinPodOrdinal(*set.Spec.Replicas, set)); err != nil {
 			return err
 		}
+
+		// end evict leader for all stores when upgrade is done
+		if err = endEvictLeaderForAllStore(m.deps, tc); err != nil {
```

Comment:
Is L762-764 still required?

Reply:
No, just to end the last scheduler more clarity.